nextcord/player.py

```diff
@@ -683,7 +695,13 @@ def _call_after(self) -> None:
 
         if self.after is not None:
             try:
-                self.after(error)
+                # Run the after function
+                after_return = self.after(error)
+
+                # If what we got back was a coroutine, submit it to
+                # the main event loop for processing
+                if asyncio.coroutines.iscoroutine(after_return):
```

Comment:
why dont use `inspect.isawaitable`?

Reply:
Jumping on this, since I missed that. Its been discussed in some help threads, but the usage of `asyncio.iscoroutinefunction` looked to be the way for cython based support as well.

Reply:
ah ok